pkg/controller/account/ec2.go

```diff
@@ -163,9 +168,9 @@ func (r *ReconcileAccount) InitializeRegion(
 }
 
 // BuildAndDestroyEC2Instances runs an ec2 instance and terminates it
-func (r *ReconcileAccount) BuildAndDestroyEC2Instances(reqLogger logr.Logger, account *awsv1alpha1.Account, awsClient awsclient.Client, ami string) error {
+func (r *ReconcileAccount) BuildAndDestroyEC2Instances(reqLogger logr.Logger, account *awsv1alpha1.Account, awsClient awsclient.Client, ami string, managedTags []awsclient.AWSTag, customerTags []awsclient.AWSTag) error {
```

Comment:
Similar to my previous comment about params, but this method has quite a few params already

Reply:
I think I'm going to leave it this way for now as we're on a time crunch to get this in, but let's create a card and we can figure out a better way to pass these through.

Maybe it would be better to overload the awsclient with the tags, then we're not even passing them into the stack at all.  Then we can just add the tags up front and just pass the client through.